### PR TITLE
Fix ruby 2.4 Fixnum deprecation warnings

### LINKED ISSUE #125: Ruby 2.4 warnings
When using gem with Ruby 2.4, it outputs lot of warnings, like

```
/home/wojtek/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/airborne-0.2.7/lib/airborne/request_expectations.rb:239: warning: constant ::Fixnum is deprecated
/home/wojtek/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/gems/airborne-0.2.7/lib/airborne/request_expectations.rb:239: warning: constant ::Bignum is deprecated
```

### PR DESCRIPTION
**Pull request by iyedb**
Fixes #125

### CODE REVIEW

**Comment by iyedb** [comment]
Should be fixed by #127 pending merge.

File: lib/airborne/request_expectations.rb [hunk 1]
```diff
@@ -267,7 +267,7 @@ def resolve_status(candidate, authority)
       candidate = Rack::Utils::SYMBOL_TO_STATUS_CODE[candidate] if candidate.is_a?(Symbol)
       case authority
       when String then candidate.to_s
-      when Fixnum then candidate.to_i
+      when 1.class then candidate.to_i
```
Comment (casiodk) [inline lib/airborne/request_expectations.rb]: why not just
`when Integer then candidate.to_i`

Comment (sethpollack) [inline lib/airborne/request_expectations.rb]: +1 for Integer

Comment (iyedb) [inline lib/airborne/request_expectations.rb]: For backward compatibility with older ruby versions.

File: lib/airborne/request_expectations.rb [hunk 2]
```diff
@@ -267,7 +267,7 @@ def resolve_status(candidate, authority)
       candidate = Rack::Utils::SYMBOL_TO_STATUS_CODE[candidate] if candidate.is_a?(Symbol)
       case authority
       when String then candidate.to_s
-      when Fixnum then candidate.to_i
+      when Integer.class then candidate.to_i
```
Comment (casiodk) [inline lib/airborne/request_expectations.rb]: `Integer.class` just gives you `Class`

It should just be `Integer`

Comment (iyedb) [inline lib/airborne/request_expectations.rb]: Right!

**Comment by iyedb** [comment]
Made the change you suggested. Thanks.

**Comment by casiodk** [comment]
@sethpollack Is it possible to merge this?

**Comment by sethpollack** [comment]
Done. Thanks!

**Comment by sletix** [comment]
I'm not sure, but I have same warnings after this merge:

```
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:239: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:239: warning: constant ::Bignum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:240: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:240: warning: constant ::Bignum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:241: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:241: warning: constant ::Bignum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:242: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:242: warning: constant ::Bignum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:243: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:243: warning: constant ::Bignum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:244: warning: constant ::Fixnum is deprecated
/Users/sletix/.rbenv/versions/2.4.0/lib/ruby/gems/2.4.0/bundler/gems/airborne-a55e8e27ea74/lib/airborne/request_expectations.rb:244: warning: constant ::Bignum is deprecated
```

**Comment by iyedb** [comment]
What is the version of your gem ?

**Comment by jondkinney** [comment]
I'm seeing those deprecation warnings bundled to master as well. It looks like either of these PRs would fix things up...not sure why they were both closed without merging though...

https://github.com/brooklynDev/airborne/pull/128
https://github.com/brooklynDev/airborne/pull/131

**Pull request merged**